sw/device/silicon_owner/bl0_tests/empty_test.c

```diff
@@ -0,0 +1,11 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <stdbool.h>
```

Comment:
this file too.

Reply:
it could! originally I was unsure if I should put in that folder or not b/c that currenlty contains a non-ottf test but I will move to that folder if you think thats ok.

Reply:
done.